packages/eas-cli/src/commands/channel/rollout.ts

```diff
@@ -391,7 +401,7 @@ export default class ChannelRollout extends EasCommand {
     if (!isRollout) {
       rolloutMutationResult = await startRolloutAsync({
         channelName,
-        branchName,
+        branchName: branchName ?? (await promptForBranchNameAsync(channelName)),
```

Comment:
working on a follow up PR to fix up how `channelName` is handled in this command as well.

Reply:
PR is here: https://github.com/expo/eas-cli/pull/930